plugins/guests/redhat/cap/configure_networks.rb

```diff
@@ -43,13 +43,18 @@ def self.configure_networks(machine, networks)
               /sbin/ifdown '#{network[:device]}' || true
 
               # Move new config into place
-              mv '#{remote_path}' '#{final_path}'
+              mv -f '#{remote_path}' '#{final_path}'
 
               # Bring the interface up
               ARPCHECK=no /sbin/ifup '#{network[:device]}'
             EOH
           end
 
+          commands << <<-EOH.gsub(/^ {14}/, '')
```

Comment:
This indent removal doesn't work. There are only 12 spaces, not 14. Looks like it was lazily copied without knowing what it does. It doesn't really make sense to have to specify the exact number of space... should just replace all leading spaces with `/^ */`.